pkg/providers/imagefamily/resolver.go

```diff
@@ -222,13 +223,19 @@ func prepareKubeletConfiguration(ctx context.Context, instanceType *cloudprovide
 	return kubeletConfig
 }
 
-func getSupportedImages(familyName *string, kubernetesVersion string, useSIG bool) []types.DefaultImageOutput {
-	imageFamily := getImageFamily(familyName, kubernetesVersion, nil)
-	return imageFamily.DefaultImages(useSIG)
+
+func getSupportedImages(familyName *string, fipsMode *v1beta1.FIPSMode, kubernetesVersion string, useSIG bool) []types.DefaultImageOutput {
```

Comment:
I'd add back this TODO:
<img width="874" height="146" alt="image" src="https://github.com/user-attachments/assets/b2d16f9f-07dd-4d54-beeb-9c5dc7ced523" />

Wasn't actually removed by you, but by Alex.

Here's plain text of the comment:
`// TODO: Options aren't used within DefaultImages, so safe to be using nil here. Refactor so we don't actually need to pass in Options for getting DefaultImage.`